tests/testthat/output/trunc_mat/POSIXlt-8-60.txt

```diff
@@ -1,12 +1,11 @@
 # A tibble: 12 x 2
-                    x         y
-               <dttm>    <dttm>
-1 2016-01-01 12:34:57 <POSIXlt>
-2 2016-01-01 12:34:58 <POSIXlt>
-3 2016-01-01 12:34:59 <POSIXlt>
-4 2016-01-01 12:35:00 <POSIXlt>
-5 2016-01-01 12:35:01 <POSIXlt>
-6 2016-01-01 12:35:02 <POSIXlt>
-7 2016-01-01 12:35:03 <POSIXlt>
-8 2016-01-01 12:35:04 <POSIXlt>
-# ... with 4 more rows
+                    x y                  
+               <dttm> <dttm>             
+1 2016-01-01 12:34:57 2016-01-01 12:34:57
```

Comment:
It's ok to show them, but they should be consistently aligned with POSIXct, and they need a different `type_sum()`